### DISCUSSION TITLE
Treat Symbols as Artboards

### DISCUSSION THREAD

**Discussion by philipp-lehmann** [ingrammicro/puzzle-publisher#34] [General]
This is an awesome plugin and I like a lot of your approaches.
Maybe I am missing something, but is there a way to include certain symbols as artboards in the prototype? 

For example: We often use a 'base' artboard and convert it to a symbol. On other arboards we place this symbol and perhaps a message on top.

Let me know if you need further clarification?

Cheers

**Comment by MaxBazarov** [comment]
Please look at this example: 
https://github.com/ingrammicro/puzzle-publisher/tree/master/tests/14.11.0

Before we also used a base artboard to show notifications. But we use another way:
1) create a symbol with some hotspot inside with "@Redirect@" string in its name.
2) create an empty "Test" artboard , configure it as "Overlay" and place the symbol inside
3) customize symbol instance and link "@Redirect@" hotspot to some real artboard which you want to see under a symbol
4) link some element to the ""Test" artboard.

Generate HTML to play with it.

<img width="1645" alt="screenshot" src="https://user-images.githubusercontent.com/18418306/111292677-7b133600-8659-11eb-9974-26170c9b03e2.png">

**Reply by philipp-lehmann** [reply]
Thanks for the instructions. This helps.
